exonum-java-binding/core/src/test/java/com/exonum/binding/core/storage/indices/BaseListIndexIntegrationTestable.java

```diff
@@ -187,6 +189,7 @@ void setReplaceAbsentElement() {
     });
   }
 
+  @Disabled //FIXME: disabled until ECR-XYZ1
```

Comment:
XYZ1?

Reply:
A placeholder, that is going to be substituted with a real task when it is created :)